app/moves/views/list-by-status.njk

```diff
@@ -2,7 +2,20 @@
 
 {% block headerGridColumnClass %}govuk-grid-column-full{% endblock %}
 
+{% block dateTitle %}
+  <ul class="govuk-list govuk-!-font-size-16 app-!-float-right app-print--hide">
```

Comment:
I think it would be good to use a consistent position of these actions like the outgoing view.

With this change the print link now appears between the caption ("Single moves") and the heading ("This 
 week") which doesn't feel correct.

Having taken a quick look at the markup I think it would be better to keep a two-thirds/one-third layout for  the header and the side actions and that the filter could be a separate block outside of this which wouldn't need any of the grid classes at all.

The fact that a block called `dateTitle` now has a print link inside it is also a bit of red flag here.

Reply:
I have reverted this commit, it will be part of a new PR at some point.